Makefile

```diff
@@ -38,13 +38,16 @@ export INSTALLATION_NAME   ?= rhmi
 export INSTALLATION_PREFIX ?= redhat-rhmi
 export USE_CLUSTER_STORAGE ?= true
 export OPERATORS_IN_PRODUCT_NAMESPACE ?= false # e2e tests and createInstallationCR() need to be updated when default is changed
+export DELOREAN_PULL_SECRET_NAME ?= integreatly-delorean-pull-secret
 
 define wait_command
 	@echo Waiting for $(2) for $(3)...
 	@time timeout --foreground $(3) bash -c "until $(1); do echo $(2) not ready yet, trying again in $(4)s...; sleep $(4); done"
 	@echo $(2) ready!
 endef
 
+IMAGE_MAPPINGS?=$(shell sh -c "find manifests/ -name image_mirror_mapping")
```

Comment:
Is this required since it's also here? https://github.com/integr8ly/integreatly-operator/blob/master/make/manifest.mk#L1